enhancements/kueue/openshift-kueue.md

```diff
@@ -0,0 +1,760 @@
+- Support for GPU model servers via pod-integration
+- Namespace opt in for frameworks.
+- Metrics for Kueue in OCP monitoring.
+- [Workload Priority Classes](https://kueue.sigs.k8s.io/docs/concepts/workload_priority_class/)
+- LocalQueue, ClusterQueues, Workloads, Worklow Priorites, Admission Checks are all supported.
+- Support for deployments, pods, statefulsets.
+- [Use of resource flavors to describe heterogeous clusters](https://kueue.sigs.k8s.io/docs/concepts/resource_flavor/)
+- Use of ManagedJobsWithoutQueueName
+- Gang admission via `WaitForPodsReady` Kueue configurations.
+- Fairsharing
+
+##### Not Supported for initial GA
+
+We will not provide the following features in tech preview.
+
+- MultiKueue
+- Autoscaling
+- TopologyAwareScheduling
+- Resource Transformations
+- KueueViz
+- [VisibilityOnDemand](https://kueue.sigs.k8s.io/docs/tasks/manage/monitor_pending_workloads/pending_workloads_on_demand/)
+- Topology CRD (Topology Aware Scheduling)
+- Cohort CRD (hierachial queueing)
+
+##### Feature support post GA
+
+Each of these features are valid. We will add support for them as dedicated RFE.
+Our focus for initial phase will be the supported features above.
+
+#### Release Schedule
+
+Need to finalize this.
+
+| Kueue Operator     |  Stage       |  OCP Version   |  Kueue   |
+| ------------------ | -------      | ---------------| -------- |
+| 1.0                | GA           | 4-17-4.19      | 0.11.z   |
+| 1.1                | GA           | 4.18-4.20      | 0.12.z   |
+| 1.2                | GA           | 4.20-4.21      | 0.13.z   |
+
+Kueue releases 6 times a year, roughly. 
+They will have 2 releases per k8s version so we can take the latest version that is built with the kubernetes version that OCP comes with.
+
+There will be no tech preview for this operator and we will go to a GA release.
+
+#### Stability of APIs
+
+A goal will be to engage with upstream to promote APIs to v1 and aim to graduate these APIs to stable.
+
+Upstream wants to promote [v1beta1 to v1beta2](https://github.com/kubernetes-sigs/kueue/issues/768) in 2025.
+
+[V1 Tracking issue](https://github.com/kubernetes-sigs/kueue/issues/3476)
+
+#### Kueue Release Committment
+
+Kueue has confirmed that they will support n-2 releases with bug fixes and security patches.
+These releases will be tested on the supported versions of Kubernetes at the time.
+There are presubmits and periodics in test-grid that will provide a [signal](https://testgrid.k8s.io/sig-scheduling) of the release.
+
+#### Release Workflow
+
+The operand will carry the same release branches as Kueue.
+These release branches will be forked in openshift and we will run their unit, integration and e2e tests on the supported versions of Kubernetes
+that this release branches were tested with in upstream. To best explain this, I think an example is necessary.
+
+In v0.11, Kueue is tested with upstream CI for 1.30-1.31-1.32. We will provide e2e testing of operand for each of these releases.
```

Comment:
out of date  I think, we're not supporting 4.17 rn right?

Reply:
We are testing 4.17 right now. I think we can consider supporting it as upstream supports this for now.